component/authenticator/src/main/java/org/wso2/carbon/identity/application/authenticator/backupcode/BackupCodeAPIHandler.java

```diff
@@ -45,128 +41,96 @@
 public class BackupCodeAPIHandler {
 
     /**
-     * Retrieve backup codes for the given username.
+     * Retrieve the count of remaining backup codes for the given username.
      *
      * @param username Username of the user.
-     * @return List of backup code of the user.
+     * @return count of remaining backup code of the user.
```

Comment:
`Returns the number of backup codes remaining for the given user`
WDYT? Also shall we improve the method comment with the same meaning?

Reply:
addressed with [7d9674b](https://github.com/wso2-extensions/identity-outbound-auth-backup-code/pull/6/commits/7d9674b6bf172b43c30479b0d64fdfa6f8130373)